packages/roosterjs-editor-api/lib/utils/processList.ts

```diff
@@ -5,76 +5,78 @@ import {
     isVoidHtmlElement,
     isBlockElement,
     Browser,
-    isNodeEmpty,
-    getTagOfNode,
-    getComputedStyles,
-    applyFormat,
-    getFirstLeafNode,
+    getSelectionPath,
+    getRangeFromSelectionPath,
 } from 'roosterjs-editor-dom';
 
 const TEMP_NODE_CLASS = 'ROOSTERJS_TEMP_NODE_FOR_LIST';
-const TEMP_NODE_HTML = `<img class="${TEMP_NODE_CLASS}">`;
+const TEMP_NODE_HTML = "<img class=\"" + TEMP_NODE_CLASS + "\">";

```

Comment:
Just curious, what's the difference between `<img class="${TEMP_NODE_CLASS}">` and "<img class=\"" + TEMP_NODE_CLASS + "\">"?

Reply:
Template strings present problems on IE11. They cause javascript exceptions, so I think it's safer we remove them (unless we no longer target IE11)